### PR TITLE
[Core] Move Stat constants out of StatTracker

### PR DESCRIPTION
**Pull request by anontyro**
Split out the static objects into there own file along with slight renaming and fixing the statTracker. Let me know if there is anything that needs updating here

#3835

### CODE REVIEW

**Comment by Dambroda** [comment]
I'm not sure if this is something we want to do. I'll leave it up for some other opinions. But this also broke some integration tests so that would need to be resolved beforehand.

**Comment by Pewtro** [comment]
So the reason I made the issue was when I was adjusting StatTracker for the Diminishing Returns and got annoyed with the fact we were defining all those default buffs in there - that said we almost never work in that file so maybe it's fine. 

In theory I'm /care either way.

**Comment by anontyro** [comment]
so I've fixed the problems the integration tests had, I think it does work to pull out some of the larger objects, I guess it also depends on how much these objects grow overtime aswell

File: src/parser/shared/modules/StatTracker.js [hunk 1]
```diff
@@ -28,322 +28,14 @@ class StatTracker extends Analyzer {
...
-    [SPELLS.MIGHT_OF_THE_BLACKROCK.id]: { versatility: 411 },
-    // endregion
-  };
-
   statBuffs = {};
 
   _pullStats = {};
   _currentStats = {};
 
-  statMultiplier = {
```
Comment (emallson) [inline src/parser/shared/modules/StatTracker.js:327]: `statMultiplier` is an internal object for `StatTracker`, not a buff list. It should stay in StatTracker. the other stuff seems fine.

**Review by emallson — CHANGES_REQUESTED** [review]

File: src/parser/shared/modules/StatTracker.js (hunk 1 above)
Comment (emallson) [reply src/parser/shared/modules/StatTracker.js:327]: @anontyro just checking in. are you planning to follow up on this PR?

**Comment by Pewtro** [comment]
Closing this due to inactivity.

**Pull request closed without merging**